lib/smart_answer_flows/coronavirus-find-support.rb

```diff
@@ -0,0 +1,277 @@
+module SmartAnswer
+  class CoronavirusFindSupportFlow < Flow
+    def define
+      name "coronavirus-find-support"
```

Comment:
So this becomes the base path - we need to check with the team if we are happy with this. Ideally, if we had something different it'd prevent collisions with the existing path and make things a bit easier to transition.

Reply:
Ah!  Thanks for pointing that out @theseanything - I wondered what purpose that served.  👍